src/main/java/github/workflow/kata/Grid.java

```diff
@@ -17,7 +17,8 @@ public Grid(String gridDef) {
         String[] rows = gridDef.trim().split("\n");
         grid = new char[rows.length][];
         for (int row = 0; row < rows.length; row++) {
-            grid[row] = rows[row].trim().toCharArray();
+            // replace all chars that is not . with *, thus adding support for live cells to be any character.

```

Comment:
Why would you add this? You have no reason to. And if you First are going down this road why would this approach be the best? For instance you could make: * and . Configurable by having a two global chars.

Reply:
This also opens up for the possibility to mix and match many different charscters which would male this weird

Reply:
It was a feature I was using when counting neighbours. thought it would be a nice feature, but as you say, it opens up for
strangeness:
```
...
.🐛.
...
```

Reply:
Will kill it of in one of the other features I am working on.